tests/test_status.py

```
@@ -134,6 +135,21 @@ def test_status_provider_filter_and_summary(state_env, tmp_path, capsys):
     assert list(summary_data["runSummary"].keys()) == ["sync drive"]
 
 
+def test_status_provider_filter_respects_runs_limit(state_env, capsys):
```

Ruff (0.15.9)

[warning] Unused function argument: `state_env`

(ARG001)